src/components/editor/property/InputLiteral.jsx

```diff
@@ -74,19 +88,30 @@ const InputLiteral = (props) => {
     error = props.errors.join(',')
   }
 
+  const handleBlur = () => {
+    if (!props.shouldShowDiacritic) addItem()
+  }
   return (
     <div className={groupClasses}>
-      <input
-            required={required}
-            className="form-control"
-            placeholder={props.propertyTemplate.propertyLabel}
-            onChange={event => setContent(event.target.value)}
-            onKeyPress={handleKeypress}
-            onBlur={addItem}
-            value={content}
-            disabled={disabled}
-            ref={inputLiteralRef}
-      />
+      <div className="input-group">
+        <input
+              required={required}
+              className="form-control"
+              placeholder={props.propertyTemplate.propertyLabel}
+              onChange={event => setContent(event.target.value)}
+              onKeyPress={handleKeypress}
+              onBlur={handleBlur}
+              value={content}
+              disabled={disabled}
+              ref={inputLiteralRef}
+              id={props.reduxPath.join('')}
```

Comment:
Would prefer including a prefix (e.g., `literal-`) to avoid id clashes.

Reply:
Done.